### PR TITLE
Changed Article and Summary section info to a hoverable

### PR DESCRIPTION
**Pull request by MridulTi**
### Describe the PR
Solved the Issue #363

- [x] SSOC Participant

I asked to change the article and summary of books section from embedded webpages to a still image which on hover shows some information about the image with the link to the original page to look into.

### Original Screenshot
image

### Updated Screenshot
image
On HOVER:-
image

### CODE REVIEW

**Review by imaaquibali — APPROVED** [review]

**Comment by imaaquibali** [comment]
@MridulTi Brother on hovring the section just show the title of the section remove the summery and also  add **Read More** **button** on every section.  It's look good.

**Comment by MridulTi** [comment]
> **ead More** **b**

So, let me make sure if this is what you mean, you mean to remove the summary on hover, and also do you want the Read More button on every hover or on Every Section, cause there is no link available that I can attach to Readmore for every section there?

**Comment by Susmita-Dey** [comment]
@MridulTi Why there's a horizontal scrolling here?
image


__
[sema-logo](https://semasoftware.com/gh) &nbsp;**Summary:** :hammer_and_wrench: This code needs a fix

**Comment by MridulTi** [comment]
> @MridulTi Why there's a horizontal scrolling here? image
> 
> __ [sema-logo](https://semasoftware.com/gh) [ sema-logo ](https://semasoftware.com/gh) [ ](https://semasoftware.com/gh)  **Summary:** 🛠️ This code needs a fix

Its for smaller screens so that the screen doesn't look too clogged up which was the reason this issue was raised my me .

**Comment by Susmita-Dey** [comment]
@MridulTi Instead of scrolling, you can shift some items down by using grid

**Comment by MridulTi** [comment]
> @MridulTi Instead of scrolling, you can shift some items down by using grid

Ok, then I will Apply it right now.

**Comment by MridulTi** [comment]
> @MridulTi Instead of scrolling, you can shift some items down by using grid
Does this look right?
image
I changed it so please view it through preview

**Comment by Susmita-Dey** [comment]
@MridulTi Your changes are not reflected in the deploy preview. What shall I do?

__
[sema-logo](https://semasoftware.com/gh) &nbsp;**Summary:** :question: I have a question&nbsp; | &nbsp;**Tags:** Not reusable

**Comment by MridulTi** [comment]
> @MridulTi Your changes are not reflected in the deploy preview. What shall I do?
> 
> __ [sema-logo](https://semasoftware.com/gh) [ sema-logo ](https://semasoftware.com/gh) [ ](https://semasoftware.com/gh)  **Summary:** ❓ I have a question  |  **Tags:** Not reusable

What? But I can clearly see my changes in the deploy preview
Here is the video link I recorded just now showing that
https://drive.google.com/file/d/1CDnRTsraunaT9_6SSs65qzh-c0iOZ_W9/view?usp=sharing

**Comment by Susmita-Dey** [comment]
@MridulTi I got it. Your screen size is bigger taht's why it's visible to you.

**Review by Susmita-Dey — CHANGES_REQUESTED** [review]
Pull the recent changes once again and make the section responsive for all screen size as the articles is disappearing at a certain screen size.
image

Kindly fix it.

**Comment by MridulTi** [comment]
> Pull the recent changes once again and make the section responsive for all screen size as the articles is disappearing at a certain screen size. image
> 
> Kindly fix it.

I fixed it kindly check it.

**Comment by Susmita-Dey** [comment]
@MridulTi Can you share a video for that as your changes are not reflected in the deploy preview?

__
[sema-logo](https://semasoftware.com/gh) &nbsp;**Summary:** :question: I have a question&nbsp; | &nbsp;**Tags:** Not reusable

**Comment by MridulTi** [comment]
> @MridulTi Can you share a video for that as your changes are not reflected in the deploy preview?
> 
> __ [sema-logo](https://semasoftware.com/gh) [ sema-logo ](https://semasoftware.com/gh) [ ](https://semasoftware.com/gh)  **Summary:** ❓ I have a question  |  **Tags:** Not reusable

Sure , Here is the link https://drive.google.com/file/d/154FSUa_aYystTZcbKhTrRl9_RcpqIE0M/view?usp=sharing

File: html/readingTherapy.html [hunk 1]
```diff
@@ -53,7 +53,47 @@ <h3 id="subtitle">Read Articles, Motivational Quotes and listen to summaries of
...
+              <button class="butn-red"><a href="https://ahigherthought.com/the-best-way-to-refresh-mind-body-soul/" target="_blank">Read More</a></button>
+      
+            </div>
+          </div>
+          <div class="collage-card">
+            <img src="../images/31inspiration.jpg"></img>
+            <div class="project-hov">
+              <h2 class="text">31 Best Inspirational & Motivational Stories</h2>
+              
+              <button class="butn-red"><a href="https://www.developgoodhabits.com/inspirational-stories/" target="_blank">Read More</a></button>
```
Comment (AliReza1083) [inline html/readingTherapy.html:72]: Can we change the class name `butn-red` to `btn-red`?

Most of the developer use btn, and easy to read.

File: css/readingTherapy.css [hunk 2]
```diff
@@ -80,6 +80,363 @@ color: #fff;
...
+  overflow:hidden;
+}
+.collage-card{
+  position: relative;
+  background-color: #3AB0FF;
+  width: 100vw;
+  height: 45vh;
+  overflow: hidden;
+}
+.collage-card>div>img{
```
Comment (AliReza1083) [inline css/readingTherapy.css:98]: It'd be great, If we add space between them.

**Review by AliReza1083 — APPROVED** [review]
Nice Work...
The result of My Code Review 👇

**Comment by Susmita-Dey** [comment]
Merge conflicts

**Pull request closed without merging**